routing_model/index.html

```diff
@@ -714,12 +714,34 @@ <h2>Process Adult/Elder promotion and demotion needed from last check</h2>
       LoopEnd --> LoopStart
   </div>
 
-  <h2>ProcessMerge Sub-routine</h2>
+  <h2>Handling merges</h2>
   <div class=description>
   <p>Vote for ParsecOurMerge, and take over handling any ParsecNeighbourMerge.<br/>
      Complete when one merge has completed, and a NewSectionInfo is consensused.<br/>
-     If Multi stage merges are required, they will require calling this function again<br/>
+     If multi-stage merges are required, they will require calling this function again<br/>
+     While in this sanctuary, our SectionInfo shall not be disturbed by elder/adult changes.
```

Comment:
adult may change: only elder do not change.

Reply:
I'll reword. I meant change as in going online/offline.

Reply:
no, they may go online offline (including elder), it is ok, they just not loose/gain elder status

Reply:
Oh. I see. You mean it's Ok if adults change during that time frame. It is. Now with the Online/Offline flow, it could actually happen.

Reply:
Yes, and it is even ok for an Elder to be voted offline, it will just not be removed from Parsec or the section until after the merge.